datalad/distribution/create_sibling.py

```diff
@@ -202,6 +203,12 @@ def _create_dataset_sibling(
         shared = CreateSibling._get_ds_remote_shared_setting(
             delayed_super, name, ssh)
 
+    if group:
+        # Either it existed before or a new directory for a repo, set its
+        # group to a desired one if was provided
+        ssh("chgrp -R {} {}".format(
+            group,
```

Comment:
While not advisable or something we're likely to encounter, it seems like group can contain characters that would need to be shell escaped (that was based on quickly googling, so I may be wrong here).  Should we use `sh_quote` here to?

Reply:
I guess it shouldn't hurt, will add now